filecoin/testutils.go

```diff
@@ -34,12 +34,12 @@ func testBlockHeader() *BlockHeader {
 		},
 		Parents:               []cid.Cid{c, c},
 		ParentMessageReceipts: c,
-		BLSAggregate:          &crypto.Signature{Type: crypto.SigTypeBLS, Data: []byte("bls signature")},
+		BLSAggregate:          &crypto.Signature{Type: crypto.SigTypeSecp256k1, Data: []byte("secp256k1 signature")},
```

Comment:
For accuracy's sake can you leave this as BLS? it doesn't matter in this test but this should be a BLS signature and same for below.